curl-and-grep/curl-and-grep.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: curl-and-grep
+spec:
+  inputs:
+    params:
+    - name: url
+      description: URL to be curl'ed.
+    - name: text
+      description: text to grep
+      default: "HTTP"
+    - name: buffer
+      description: file to save output
+      default: '/workspace/buffer.txt'
+  steps:
+  - name: grep
```

Comment:
step names are mixed up? the grep step does a curl, the curl step does a grep

Reply:
you're right